sys/vm/vm_map.h

```diff
@@ -105,6 +105,7 @@ struct vm_map_entry {
 	struct vm_map_entry *right;	/* right child in binary search tree */
 	vm_offset_t start;		/* start address */
 	vm_offset_t end;		/* end address */
+	vm_offset_t reservation;	/* VM reservation ID */
```

Comment:
"/* VM reservation ID (lowest address in original request) */" or such ?

Reply:
I'd originally thought that I might want reservation IDs to come from some other source, but with later work it become obvious that I really do want it to be the VA.